README.md

```diff
@@ -800,7 +909,11 @@ our team.
     monitoring of our team’s competitiveness.
 
 For additional information regarding our implementation plan, please
-refer to ![Report.pdf](ACTL4001_Group_Z_Case_Report%20Submission.pdf).
+refer to ![](ACTL4001_Group_Z_Case_Report%20Submission.pdf)
```

Comment:
Can you please have this instead? ![Report.pdf](ACTL4001_Group_Z_Case_Report%20Submission.pdf).